pyproject.toml

```diff
@@ -3,49 +3,57 @@ name = "pylint-nautobot"
 version = "0.2.1"
 description = "Custom Pylint Rules for Nautobot"
 authors = ["Cristian Sirbu <cristian.sirbu@networktocode.com>", "Leo Kirchner <leo.kirchner@networktocode.com>"]
+license = "Apache-2.0"
 readme = "README.md"
+homepage = "https://github.com/nautobot/pylint-nautobot"
+repository = "https://github.com/nautobot/pylint-nautobot"
+keywords = ["nautobot", "pylint"]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+packages = [
+    { include = "pylint_nautobot" },
+]
 
 [tool.poetry.dependencies]
-python = "^3.7"
-pylint = "^2.13"
-pyyaml = "^6.0"
-importlib-resources = "^5.12.0"
-tomli = "^2.0.1"
+importlib-resources = ">=6.0.0"
+pylint = ">=2.17.5"
+python = ">=3.8,<3.12"
+pyyaml = ">=6.0.1"
+toml = ">=0.10.2"
 
 [tool.poetry.group.dev.dependencies]
-black = "^23.3.0"
-pydocstyle = "^6.3.0"
-flake8 = "^3.9.2"
-bandit = "^1.7.5"
-invoke = "^2.0.0"
-toml = "^0.10.2"
-yamllint = "^1.30.0"
-pytest = "^7.3.0"
-# Rendering docs to HTML
-mkdocs = "~1.4.3"
-# Material for MkDocs theme
+Markdown = "*"
+bandit = "*"
+black = "*"
+coverage = "*"
+flake8 = "*"
+invoke = "*"
+ipython = "*"
+mkdocs = "1.5.2"
 mkdocs-material = "9.1.15"
-# Render custom markdown for version added/changed/remove notes
 mkdocs-version-annotations = "1.0.0"
-# Automatic documentation from sources, for MkDocs
-mkdocstrings = "0.22.0"         # Last version with python 3.7 support
-mkdocstrings-python = "1.1.2"   # Last version with python 3.7 support
-griffe = "0.30.1"               # Last version with python 3.7 support
-# Django and nautobot have to be installed for the inference in the tests of the class hierarchy to work correctly
-django = "^3.2.19"
-nautobot = "^1.5.19"
-
-[tool.pylint-nautobot]
-# Normally this configuration goes into the apps that should be linted, but this is required so that the checkers
-# activate correctly during unit testing.
-supported_nautobot_versions = [
-    "1",
-    "2"
+mkdocstrings = "0.22.0"
```

Comment:
I think there's some duplication here with the main package dependencies, could you please clean it up?

Reply:
Removed